src/main/java/net/snowflake/client/jdbc/SnowflakePreparedStatementV1.java

```diff
@@ -133,6 +140,12 @@ private void describeSqlIfNotTried() throws SQLException {
     if (!alreadyDescribed) {
       try {
         this.preparedStatementMetaData = sfBaseStatement.describe(sql);
+        if (preparedStatementMetaData != null
+            && !preparedStatementMetaData.isArrayBindSupported()) {
+          logger.debug(
```

Comment:
Is debug a good level?

Reply:
yeah, I believe so -we can make it higher if necessary in the future, but normally to check what's happening you are switching to at least debug to investigate